docker/README.md

```diff
@@ -0,0 +1,149 @@
+  - [Checking the container logs](#checking-the-container-logs)
+  - [Login into the running container to use **KubeStellar** from inside the container](#login-into-the-running-container-to-use-kubestellar-from-inside-the-container)
+  - [Teardown and cleanup](#teardown-and-cleanup)
+
+## Install the pre-requisites
+
+Install `nake`:
+
+```bash
+sudo apt install -y make
+```
+
+Install `docker`:
+
+```bash
+sudo mkdir -p /etc/apt/keyrings
+curl -fsSL https://download.docker.com/linux/ubuntu/gpg | sudo gpg --dearmor -o /etc/apt/keyrings/docker.gpg
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/etc/apt/keyrings/docker.gpg] https://download.docker.com/linux/ubuntu $(lsb_release -cs) stable" | sudo tee /etc/apt/sources.list.d/docker.list > /dev/null
+sudo apt update
+sudo apt install -y docker-ce docker-ce-cli containerd.io docker-compose-plugin
+sudo usermod -aG docker $USER
+```
+
+Or `podman` (multi-arch `buildx` option will not be available):
+
+```bash
+sudo apt install -y podman
+alias docker=podman
+```
+
+## Build a local container image
+
+Build a local container image `kubestellar` for the host architecture using the latest release of **KubeStellar** with the command:
+
+```bash
+make build
+```
+
+A desired version of **KubeStellar** can be specified by using the optional `KUBESTELLAR_VERSION` argument, such as `KUBESTELLAR_VERSION=v0.3.1`.
+
+The **KubeStellar** version is used to tag the container image, as shown below:
+
+```bash
+$ docker images
+REPOSITORY    TAG       IMAGE ID       CREATED          SIZE
+kubestellar   v0.3.1    0425aee2cda6   8 minutes ago    617MB
+kubestellar   v0.3.0    76d714261c6e   14 minutes ago   617MB
+```
+
+## Build and push a multi-architecture image
+
+Note that `docker buildx` is required for this option.
+
+Make sure to login into the registry with the desired user:
+
+```bash
+docker login -u <user> <registry>
+```
+
+Build and push a multi-architecture image using the command:
+
+```bash
+make buildx IMG=<registry>/<organization>/<image>
+```
+
+A desired version of **KubeStellar** can be specified by using the optional `KUBESTELLAR_VERSION` argument, such as `KUBESTELLAR_VERSION=v0.3.1`, Default value: the latest release version of **KubeStellar**.
+
+The list of desired target platforms can be specified by using the optional `PLATFORMS` argument. Default value; `PLATFORMS=linux/amd64,linux/arm64,linux/ppc64le`.
+
+## Run the container and access KubeStellar from the host OS
+
+Spin up a local or remote container image with the command:
+
+```bash
+make run
+export KUBECONFIG=${HOME}/.kcp/admin.kubeconfig
+export PATH=$PATH:${HOME}/kcp:${HOME}/kubestellar/bin
+```
+
+The following arguments can be used:
+
+- `IMG=[<registry>/<organization>/]<image>` select the container image, the default vakue is the local `kubestellar` image
+- `KUBESTELLAR_VERSION=[<version>|latest]`, select the image tag, the default value is the latest version of **KubeStellar**

```

Comment:
Does `latest` really work here?  I do not see the tag `latest` being applied in the builds.

Reply:
not really cleanly, I have removed it